helm/sematic-server/templates/serviceaccount.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if .Values.serviceAccount.create }}
```

Comment:
I couldn't succesfully deploy into dev1 without this. We know a customer wants to use their own existing SA without letting the chart create it, so we might as well keep this change.

Reply:
yes, i had to add this in my own branch (unmerged) as well, so this is good to keep.